Tests/EventListener/FinishControllerSpanSubscriberTest.php

```diff
@@ -7,35 +7,33 @@
 use Auxmoney\OpentracingBundle\EventListener\FinishControllerSpanSubscriber;
 use Auxmoney\OpentracingBundle\Internal\TracingId;
 use Auxmoney\OpentracingBundle\Service\Tracing;
-use Auxmoney\OpentracingBundle\Tests\Mock\EventWithNoResponse;
-use Auxmoney\OpentracingBundle\Tests\Mock\EventWithResponse;
-use Auxmoney\OpentracingBundle\Tests\Mock\EventWithResponseAndReflectionError;
 use PHPUnit\Framework\TestCase;
 use Prophecy\Argument;
 use Prophecy\PhpUnit\ProphecyTrait;
-use Psr\Log\LoggerInterface;
 use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Event\ResponseEvent;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
 
 class FinishControllerSpanSubscriberTest extends TestCase
 {
     use ProphecyTrait;
 
     private $tracingId;
```

Comment:
You could also use typed class properties in here. Also applies for the other tests

Reply:
True, done.